internal/model/view.go

```diff
@@ -11,20 +12,261 @@ import (
 	"github.com/epilande/codegrab/internal/utils"
 )
 
+// View renders the entire UI model.
 func (m Model) View() string {
+	// If help is shown, render a simple help view.
 	if m.showHelp {
 		header := ui.GetStyleHeader().Render("❔ Help Menu")
-		content := ui.GetStyleBorderedViewport().Render(m.viewport.View())
-		footer := ui.GetStyleHelp().Render("Exit: esc")
-		return header + "\n" + content + "\n" + footer
+		headerHeight := lipgloss.Height(header)
+		footerText := "Exit: esc" // Example footer text for height calculation
+		footer := ui.GetStyleHelp().Render(footerText)
+		footerHeight := lipgloss.Height(footer)
+		availableHeight := m.height - headerHeight - footerHeight
```

Comment:
We need to account for the border here to fix the help screen viewport

```suggestion
		availableHeight := m.height - headerHeight - footerHeight - (2 * ui.BorderSize)
```